modules/javafx.graphics/src/main/native-glass/mac/GlassTouches.m

```diff
@@ -176,6 +176,11 @@ + (void)terminate
 
 - (id)init
 {
+    BOOL useEventTap = YES;
```

Comment:
I think that if you make useEventTap static, you don't need the additional tests in the other functions.
init is always the first one to be called.

Reply:
OK, I'll push an update for this.